src/classes/OPP_OpportunityNaming_BATCH.cls

```diff
@@ -1,10 +1,8 @@
 /*
     Copyright (c) 2015, Salesforce.org
     All rights reserved.
-    
```

Comment:
These blank lines should probably be here to remain consistent with our copyright notice in all the other classes.

Reply:
Whoops will fix it.